Number_types/include/CGAL/Interval_nt.h

```diff
@@ -1141,16 +1141,20 @@ namespace INTERN_INTERVAL_NT {
       // it helps significantly, it might even hurt by introducing a
       // dependency.
     }
-#else
+#else // no __AVX512F__
     // TODO: Alternative for computing CGAL_IA_SQRT_DOWN(d.inf()) exactly
     // without changing the rounding mode:
     // - compute x = CGAL_IA_SQRT(d.inf())
     // - compute y = CGAL_IA_SQUARE(x)
     // - if y==d.inf() use x, else use -CGAL_IA_SUB(CGAL_IA_MIN_DOUBLE,x)
+#ifdef CGAL_ALWAYS_ROUND_TO_NEAREST
```

Comment:
`#elif` is usually nicer than nested `#if`.

Reply:
I'm happy to change it, but was concerned that the context of the comment about the alternative approach would become confused.

Reply:
I think this is obsoleted by the abstraction into IA_sqrt_toward_zero.